src/cf-typography/usage.md

```diff
@@ -316,61 +265,89 @@ attached to `.meta-header_left` and `.category-slug` is attached to `.meta-heade
 
 #### Default meta header
 
-<div class="meta-header">
-    <span class="meta-header_right date">
-        Nov 4, 2013
-    </span>
-    <a href="#" class="meta-header_left category-slug">
-        <span class="cf-icon cf-icon-credit-card"></span>
-        Consumer finance
-    </a>
+<div class="m-meta-header">
+    <div class="m-meta-header_right">
+        <span class="date">
+            Nov 4, 2013
+        </span>
+    </div>
+    <div class="m-meta-header_left">
+        <a href="#" class="m-meta-header_left category-slug">
+            <span class="cf-icon cf-icon-credit-card"></span>
+            Consumer finance
+        </a>
+    </div>
 </div>
 
 ```
-<div class="meta-header">
-    <span class="meta-header_right date">
-        Nov 4, 2013
-    </span>
-    <a href="#" class="meta-header_left category-slug">
-        <span class="cf-icon cf-icon-credit-card"></span>
-        Consumer finance
-    </a>
+<div class="m-meta-header">
+    <div class="m-meta-header_right">
+        <span class="date">
+            Nov 4, 2013
+        </span>
+    </div>
+    <div class="m-meta-header_left">
+        <a href="#" class="m-meta-header_left category-slug">
+            <span class="cf-icon cf-icon-credit-card"></span>
+            Consumer finance
+        </a>
+    </div>
 </div>
 ```
 
 #### Alternate meta header arrangements
 
 This is just another set of variations to show off what you could do.
 
-<div class="meta-header">
-    <a href="#" class="meta-header_right category-slug">
-        <span class="cf-icon cf-icon-credit-card"></span>
-    </a>
-    <span class="meta-header_left date">
-        Nov 4, 2013
-    </span>
+<div class="m-meta-header">
+    <div class="m-meta-header_left">
```

Comment:
I think the alternate usage was mostly to demonstrate flopping the date and category, but honestly, it's probably not worth showing any alternate usage. It should be pretty self-evident what can be done with it.